### ISSUE TITLE
Assignment - 03: Tableau

### ISSUE THREAD

**Issue by thanhhff** [vutung3196/DevC-Challenge-Data-Science#3]

**Comment by thanhhff** [comment]
Anh nên tổng hợp các phần nhỏ thành 1 Dashboard nhé!
Ví dụ anh có thể tham khảo: https://public.tableau.com/profile/nguyen.minh.anh#!/vizhome/covid19_15924716772030/Dashboard?publish=yes

**Comment by vutung3196** [comment]
anh tổng hợp vào 1 file rồi ý, em có thể xem ở link này: https://public.tableau.com/profile/tung3828#!/vizhome/covid19-workbook_15951500157700/Covid-19Dashboard?publish=yes

**Comment by thanhhff** [comment]
Yeah, oke a. Hôm qua em download file kia không có cái này, ổn rồi nhé a :D
